addon/components/nav-section-action.js

```diff
@@ -3,5 +3,11 @@ import layout from '../templates/components/nav-section-action'
 
 export default Ember.Component.extend({
   classNames: ['nav-column', 'nav-section-action'],
-  layout
+  goBack: function () {},
```

Comment:
No need to provide a default, just to a null check around `back` (defaults have odd behaviors with components)